Drivers/Inc/Display.h

```diff
@@ -10,39 +10,54 @@
 
 #include "common.h"
 #include "config.h"
+#include "Minions.h"
 
-/*
- * Initialize the Nextion display
- */
-void Display_Init();
+#define MAX_ARGS 2
 
-/**
- * Value setting subroutines
- */
-ErrorStatus Display_SetMainView(void);
+typedef enum{
+	STARTUP=0,
+	INFO,
+	FAULT
+} Page_t;
+
+typedef struct{
+	char* compOrCmd;
+	char* attr;
+	char* op;
+	uint8_t numArgs;
+	bool argTypes[MAX_ARGS];	// TRUE for integers, FALSE for strings
+	union{
+		char* str;
+		uint8_t num;
```

Comment:
Should probably be a 32 bit integer, since many of our values are bigger than 8 bits. The union is already 32 bits in size anyways, due to the char *.

Reply:
I made sure that none of the numbers we were sending over are >255. The percentage we're displaying is an integer from 0-100 and the voltage is 0-12.

Reply:
What about the velocity? I thought we were sending it with a granularity of 0.1 mph. Or is that also changed? Either way, we have a 32-bit space for an integer there, so we can have it as an int32_t for the future.

Reply:
Ooh I forgot about the velocity. I'll change it to a 32 bit then.